orchestrator/crates/orchestrator/src/tests/jobs/snos_job/mod.rs

```diff
@@ -46,10 +46,16 @@ async fn test_verify_job(#[from(default_job_item)] mut job_item: JobItem) {
     // Update job_item to use the proper metadata structure for SNOS jobs
     job_item.metadata.specific = JobSpecificMetadata::Snos(SnosMetadata::default());
 
-    let job_status = SnosJob.verify_job(services.config.clone(), &mut job_item).await;
+    let job_status = SnosJobHandler.verify_job(services.config.clone(), &mut job_item).await;
 
-    // Should always be [Verified] for the moment.
-    assert_eq!(job_status, Ok(JobVerificationStatus::Verified));
+    assert_eq!(
+        matches!(job_status, Ok(JobVerificationStatus::Verified)),
+        true,
+        "Job verification status should be Verified or NotVerified"
+    );
```

Comment:
You can replace this with an `assert_matches`.
```suggestion
    assert_matches!(
        job_status,
        Ok(JobVerificationStatus::Verified),
        "Job verification status should be Verified or NotVerified"
    );
```